### PR TITLE
Feature/usdc usdt usn 3pool

### PR DESCRIPTION
**Pull request by rej156**
[https://github.com/trisolaris-labs/trisolaris_core/blob/main/scripts/stableswap.ts](https://github.com/trisolaris-labs/trisolaris_core/blob/main/scripts/stableswap.ts) 

copied and trimmed tokens to 3 pool, added programmatic contract verification

### CODE REVIEW

File: scripts/setup3PoolAdminFee.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  const SwapFlashLoanFactory = await ethers.getContractFactory("SwapFlashLoan", {
+    libraries: {
+      SwapUtils: swapUtils.address,
+      AmplificationUtils: amplificationUtils.address,
+    },
+  });
+
+  const swapFlashLoan = await SwapFlashLoanFactory.connect(deployer).attach(
+    "0x458459E48dbAC0C8Ca83F8D0b7b29FEfE60c3970",
```
Comment (0xchain1) [inline scripts/setup3PoolAdminFee.ts:37]: want to confirm that these are the latest and most accurate contracts

Comment (rej156) [reply scripts/setup3PoolAdminFee.ts:37]: same as the ones in the comments below and same commented addresses as the deploy3poolStableswap.ts

File: scripts/deploy3poolStableswap.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  // await run("compile");
+  // We get the contract to deploy
+
+  // Constants
+  const [_, deployer] = await ethers.getSigners();
+  console.log(`Deploying contracts with ${deployer.address}`);
+
+  const usdcAddress = "0xb12bfca5a55806aaf64e99521918a4bf0fc40802";
+  const usdtAddress = "0x4988a896b1227218e4a686fde5eabdcabd91571f";
+  const fraxAddress = "0xE4B9e004389d91e4134a28F19BD833cBA1d994B6";
```
Comment (baboobhaiya) [inline scripts/deploy3poolStableswap.ts:23]: remove, frax stuff

File: scripts/setup3PoolAdminFee.ts (hunk 1 above)
Comment (baboobhaiya) [reply scripts/setup3PoolAdminFee.ts:37]: can we add these contract addresses in the `constants.ts` file https://github.com/trisolaris-labs/trisolaris_core/blob/main/scripts/constants.ts#L46 like this and import from there?

File: scripts/setup3PoolAdminFee.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  );
+  console.log(`swapFlashLoan connected and attached at "${swapFlashLoan.address}"`);
+
+  // Set Fee Address to deployer
+  await swapFlashLoan.setFeeAddress(deployer.address);
```
Comment (baboobhaiya) [inline scripts/setup3PoolAdminFee.ts:42]: no need to do this: https://github.com/trisolaris-labs/trisolaris_core/blob/main/contracts/stableswap/Swap.sol#L189

we do this anywats when we initialize the contract

**Review by baboobhaiya — APPROVED** [review]

**Pull request merged**